live-examples/css-examples/images/image-rendering.css

```diff
@@ -0,0 +1,5 @@
+#example-element {
+    height: 500px;
+    object-fit: cover;
+    border: 1px solid red;
```

Comment:
This is a tricky one indeed, but a really need new CSS thing that is on the horizon. My only suggestion here would be to remove the border, and set the `height` to `480px`

Reply:
Fixed, thank you!